schema/information_schema_1-2-0.json

```diff
@@ -0,0 +1,241 @@
+  "description": "Information JSon schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "type",
+    "version",
+    "instance_id",
+    "instance_type",
+    "running",
+    "timestamp",
+    "validity_duration"
+  ],
+  "properties": {
+    "type": {
+      "description": "type of server",
+      "type": "string",
+      "enum": [
+        "broker"
+      ]
+    },
+    "version": {
+      "type": "string",
+      "description": "json message format version",
+      "const": "1.1.99"
+    },
+    "instance_id": {
+      "description": "unique id all other the world for a server",
+      "type": "string"
+    },
+    "instance_type": {
+      "description": "type of instance",
+      "type": "string",
+      "enum": [
+        "edge",
+        "central"
+      ]
+    },
+    "central_instance_id": {
+      "description": "unique id all other the world for the central server if the instance is of type edge",
+      "type": "string"
+    },
+    "running": {
+      "description": "state of the server: true id running, false else.",
+      "type": "boolean",
+      "default": false
+    },
+    "timestamp": {
+      "description": "generation timestamp of the information, millisecond since epoch (1970/01/01)",
+      "type": "integer"
+    },
+    "validity_duration": {
+      "description": "validity period of the information relatively to the timestamp in seconds",
+      "type": "integer"
+    },
+    "public_ip_address": {
+      "description": "list of public IP address",
+      "type": "array",
+      "examples": [
+        "161.05.21.166",
+        "190.14.19.50",
+        "127.0.0.1"
+      ],
+      "items": {
+        "description": "public IP address",
+        "type": "string"
+      }
+    },
+    "mqtt_ip": {
+      "description": "list of MQTT IP address and port separated by a colon",
+      "type": "array",
+      "items": {
+        "description": "MQTT IP address and port (classically 1883) separated by a colon",
+        "type": "string",
+        "examples": [
+          "84.188.43.159:1883",
+          "172.116.12.182:11883"
+        ]
+      }
+    },
+    "mqtt_tls_ip": {
+      "description": "list of MQTT IP address and TLS port separated by a colon",
+      "type": "array",
+      "items": {
+        "description": "MQTT IP address and TLS port (classically 8883) separated by a colon",
+        "type": "string",
+        "examples": [
+          "84.188.43.159:8883",
+          "172.116.82.124:18883"
+        ]
+      }
+    },
+    "http_proxy": {
+      "description": "list of HTTP IP address and port separated by a colon",
+      "type": "array",
+      "items": {
+        "description": "HTTP IP address and port (classically 8080) separated by a colon",
+        "type": "string",
+        "examples": [
+          "84.188.43.159:8080",
+          "172.116.82.124:1280"
+        ]
+      }
+    },
+    "ntp_servers": {
```

Comment:
"ntp_server" or "ntp_server_list"... or put a "s" on all other fields : "mqtt_tls_ips", "public_ip_addresses"...
I would prefer the explicit json type format : "ntp_server_list", "mqtt_tls_ip_list", "public_ip_address_list",...

Reply:
This what it was in version 1.1.0. I don't want that we do in-depth changes, but
just be as compatible as possible.

Reply:
So do we create an issue for?